app/Http/Requests/SkillsRequest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Rules\DniNieRule;
+use Illuminate\Contracts\Validation\Validator;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Http\Exceptions\HttpResponseException;
+
+class SkillsRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+
+
+	protected function prepareForValidation()
+	{
+		$this->merge([
+			'skills' => json_encode($this->skills)
+		]);
+	}
+
+    /**
+     * Get the validation rules that apply to the request.
```

Comment:
Remove unnecessary information in comments

Reply:
Done